### PR TITLE
Remove environment variables that are not working & add default DB connection name

### PR DESCRIPTION
**Pull request by lciolecki**
## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

### CODE REVIEW

File: designer/server/src/main/resources/defaultDesignerConfig.conf [hunk 1]
```diff
@@ -21,6 +21,10 @@ db {
   minimumIdle: 1
   #has to be lower <= maxConnections
   numThreads: 5
+  properties: {
+    # PostgreSQL specific
+    "ApplicationName": "Nussknacker"
```
Comment (arkadius) [inline designer/server/src/main/resources/defaultDesignerConfig.conf:26]: What is it? Why it is in defaultDesignerConfig?

Comment (lciolecki) [reply designer/server/src/main/resources/defaultDesignerConfig.conf:26]: To niczego nie psuje. Jak opisano w komentarzu powyżej: jest to bardziej specyficzne dla postgresql. Aby uniknąć nadpisywania konfiguracji DB, gdy uruchamiamy NU z PostgreSQL jako persistence. Ta właściwość jest używana do zadeklarowania / nazwania połączenia wykonanego przez Nussknacker do DB.

File: docs/Changelog.md [hunk 2]
```diff
@@ -55,15 +55,15 @@
...
-  helper.
+* [#6623](https://github.com/TouK/nussknacker/pull/6623) Added `sortedAscBy` and `reverse` functions to `#COLLECTION` helper
 * [#6650](https://github.com/TouK/nussknacker/pull/6650) Added a workaround for situations when an updated scenario with Kafka source started reading from the earliest offsets
 * [#6586](https://github.com/TouK/nussknacker/pull/6586) For now on, the SQL enricher automatically converts types as shown below:
   * java.sql.Array -> java.util.List
   * java.sql.Time -> java.time.LocalTime 
   * java.sql.Date -> java.time.LocalDate
   * java.sql.Timestamp -> java.time.Instant
   * java.sql.Clob -> java.lang.String
+* [#6656](https://github.com/TouK/nussknacker/pull/6656) Remove environment variables that are not working & add default DB connection name
```
Comment (arkadius) [inline docs/Changelog.md:66]: Let's write:
> - Remove not working `MODEL_CLASS_PATH`  environment variable
> - (another point) Add default DB connection name

File: docs/MigrationGuide.md [hunk 3]
```diff
@@ -68,6 +68,8 @@ To see the biggest differences please consult the [changelog](Changelog.md).
     * java.sql.Date -> java.time.LocalDate
     * java.sql.Timestamp -> java.time.Instant
     * java.sql.Clob -> java.lang.String
+* [#6656](https://github.com/TouK/nussknacker/pull/6656) Remove environment variables that are not working & add default DB connection name
```
Comment (arkadius) [inline docs/MigrationGuide.md:71]: We don't need any migration because it didn't work so nobody used it

**Review by arkadius — APPROVED** [review]

**Pull request merged**